app/models/case_log.rb

```diff
@@ -723,4 +700,57 @@ def soft_value_for_period(value)
   def upcase_and_remove_whitespace(string)
     string.present? ? string.upcase.gsub(/\s+/, "") : string
   end
+
+  def housingneeds_present?
+    housingneeds.present?
+  end
```

Comment:
Do you need a method at all here? Can you get away with just using `if :housingneeds?`?

Reply:
Ah, I think I added it because the initial values for housingneeds contained 0 as an option, but it doesn't anymore. Good spot!